zebrad/Cargo.toml

```diff
@@ -64,22 +63,12 @@ default-release-binaries = ["default", "sentry"]
 # Indexer support
 indexer = ["zebra-rpc/indexer-rpcs", "zebra-state/indexer"]
 
-# Mining RPC support
-getblocktemplate-rpcs = [
-    "zebra-rpc/getblocktemplate-rpcs",
-    "zebra-consensus/getblocktemplate-rpcs",
-    "zebra-state/getblocktemplate-rpcs",
-    "zebra-node-services/getblocktemplate-rpcs",
-    "zebra-chain/getblocktemplate-rpcs",
-]
-
```

Comment:
I think it might be better to leave the feature (i.e. just leave a `getblocktemplate-rpcs = []` here) so that existing deployment code doesn't break (e.g. anyone who has `cargo build --features getblocktemplate-rpcs` hardcoded somewhere)

(and also document that it's an empty, no longer used feature)

We could do the same for the other crates, but I think that they don't have many users as zebra itself and we can get away with just removing the feature in them. Up to you.

Reply:
I'm happy to make this change, but will it not create a precedent where we will have to keep every feature we ever use forever? Simply nuking it and announcing it in the changelog seems better than shadowing it.

Reply:
We can get rid of it when we make a 3.0.0 release, after we batch some more breaking changes to do together. (we should list them all in an issue so we don't forget...) I think we should try to minimize breakage for zebra users when possible.

Reply:
Done in [837a022](https://github.com/ZcashFoundation/zebra/pull/9401/commits/837a022fdc1472061ba41736621d08609bc1b0d9).